content/en/blog/_posts/2024-05-13-Gateway-API-v1.1/index.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog
+title: "Gateway API v1.1: Feature Graduation and Other Updates"
+date: 2024-05-13T19:00:00-08:00
+slug: gateway-api-v1-1
+---
+
+![Gateway API Logo](gateway-api-logo.png)
+**Authors:** 
+GRPCRoute - Richard Belleville (Google), 
+ParentRef Port - Frank Budinsky (IBM),
+Gateway Client Certificate Validation - Arko Dasgupta (Tetrate),
+GAMMA - John Howard (Solo.io),
+Conformance Profiles and Reports - Mattia Lavacca (Kong), Christine Kim (Isovalent),
+Session Persistence and BackendLBPolicy - Grant Spence (Red Hat), Gina Yeh (Google),
+TLS Terminology Clarifications - Candace Holman (Red Hat),
+and other reviewer and release note contributors.
+
+In the wake of the GA release of Gateway API last October, the Kubernetes
+SIG Network buoyantly announces the v1.1 release
```

Comment:
```suggestion
SIG Network is happy to announce the v1.1 release
```
Maybe just me, but immediately sounded like a pun :) Buoyant -> Linkerd

Reply:
I wanted to keep the "sailing" metaphor going... "in the wake of".... "buoyantly". ⛵

Reply:
Not gonna lie, I had that thought as well, but decided I was reading too much into a nautical metaphor. 🙂 

For the record, I'm confident that no one from Buoyant will be offended if we change it. 🙂